### PR TITLE
[13.0][IMP] account_invoice_inter_company: Remove onchange_helper dependency

### PR DESCRIPTION
**Pull request by pedrobaeza**
Removes `onchange_helper` dependency and uses fancy `Form` class for the equivalent one (and even more complete and bullet-proof).

Forward-port to v13 of #218

cc @Tecnativa TT21089

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Tested in runbot :+1:

File: account_invoice_inter_company/views/account_move_views.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <record id="view_move_form" model="ir.ui.view">
+        <field
+            name="name"
+        >account.move.form - HACK: Extra fields for odoo.tests.Form() to work</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']/tree" position="inside">
+                <field name="company_id" invisible="0" readonly="0" />
```
Comment (sergio-teruel) [inline account_invoice_inter_company/views/account_move_views.xml:12]: Always visible???

**Review by sergio-teruel — APPROVED** [review]
Only a little comment

File: account_invoice_inter_company/views/account_move_views.xml (hunk 1 above)
Comment (chienandalu) [reply account_invoice_inter_company/views/account_move_views.xml:12]: It's a primary for this specific purpose, so it shouldn't be a problem, isn't it?

Comment (pedrobaeza) [reply account_invoice_inter_company/views/account_move_views.xml:12]: Indeed, although I think I wanted to put invisible in that moment and it was a typo. It doesn't matter anyway.

**Pull request merged**